src/main/java/com/rakovets/course/java/core/practice/decision_making_statements/Task11.java

```diff
@@ -40,6 +40,15 @@ public static void main(String[] args) {
     static String getSymbolType(char symbol) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return null;
+        int value = symbol;
+        String symbolType = "undefined";
+        if (value >= 48 && value <= 57) {
+            symbolType = "digit";
+        } else if ((value >= 65 && value <= 90) || (value >= 97 && value <= 122)) {
+            symbolType = "latin";
+        } else if (value >=1024 && value <= 1279) {
```

Comment:
"Violation: There isn't separating white spaces for operators OR parameters
Solution: Add white spaces"